views/application-overview.njk

```diff
@@ -0,0 +1,46 @@
+{% extends 'layout.njk' %}
+{% block content %}
+    <div class="govuk-width-container">
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-two-thirds">
+                    <h2 class="govuk-heading-l">{{name}}</h2>
+                </div>
+            </div>
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-two-thirds">
+                    <hr class="banner-line">
+                    <h2 class="govuk-heading-l">-------------------Put timeline here!!----------------</h2>
+                </div>
+                <div class="govuk-grid-column-one-third">
+                    <div>
+                        <hr class="contactus-line">
+                        <b class="govuk-heading-s">{{ i18n.pages.overviewPage.contactUs }}</b>
+                        <p class="govuk-body">{{ i18n.pages.overviewPage.askAQuestion }}</p>
+                        <p class="govuk-body">{{ i18n.pages.overviewPage.telephone }}<br><a href="#" class="govuk-link">{{ i18n.pages.overviewPage.sendMessage }}</a> </p>
+                        <p class="govuk-body">{{ i18n.pages.overviewPage.time }}<br> <a href='https://www.gov.uk/call-charges' target='_blank' class="govuk-link">{{ i18n.pages.overviewPage.charges }}</a> <br>{{ i18n.pages.overviewPage.newWindow }}</p>
+                        <hr class="contactus-line">
```

Comment:
GDS has already a separator or section break we can use please refactor with it and maybe add a modifier class 

https://design-system.service.gov.uk/styles/typography/